docs/install/grafana-setup.rst

```diff
@@ -165,12 +165,12 @@ connection is successful.
 
 .. _grafana-import-dashboard-file:
 
-Import the Omniperf dashboard file
-----------------------------------
+Import the ROCm Compute Profiler dashboard file
+-----------------------------------------------
 
 From the **Create** → **Import** page, upload the dashboard file,
 ``/dashboards/Omniperf_v{__VERSION__}_pub.json`` from the
-:doc:`Omniperf tarball <core-install>`.
+:doc:`ROCm Compute Profiler tarball <core-install>`.
```

Comment:
Will the dashboard files be renamed? https://github.com/ROCm/omniperf/pull/428/files#r1813277400